chef/cookbooks/monasca/attributes/default.rb

```diff
@@ -52,3 +49,19 @@
 default[:monasca][:log_agent][:user] = "monasca-log-agent"
 default[:monasca][:log_agent][:group] = "logstash"
 default[:monasca][:log_agent][:debug] = monasca_debug
+
+# default[:monasca][:api][:bind_host] = "*"
+# default[:monasca][:log_api][:bind_host] = "*"
+# default[:monasca][:kibana][:bind_host] = "*"
+# default[:monasca][:mariadb][:bind_host] = "-"
+# default[:monasca][:influxdb][:bind_host] = "-"
+
+default[:monasca][:ha][:enabled] = monasca_ha_enabled
+# Ports to bind to when haproxy is used for the real ports
+# Public network
+default[:monasca][:ha][:ports][:api] = 18070
+default[:monasca][:ha][:ports][:log_api] = 15607
+default[:monasca][:ha][:ports][:kibana] = 15601
+# Internal network
+default[:monasca][:ha][:ports][:mariadb] = 13306
+default[:monasca][:ha][:ports][:influxdb] = 18086
```

Comment:
Added InfluxDB Relay port but need to learn how to use `frontend` and `backend` HAProxy configuration with barclamps. For now added `FIXME` comments in code.